app/components/custom_fields/details_component.rb

```diff
@@ -32,5 +32,13 @@ module CustomFields
   class DetailsComponent < ApplicationComponent
     include OpPrimer::ComponentHelpers
     include OpTurbo::Streamable
+
+    alias_method :custom_field, :model
+
+    def has_no_items_or_projects?
```

Comment:
🟡 Can we leverage the `render?` method

Reply:
I'd say no, as I'd need to create a separate component just for the banner.